dist/alt-browser-with-addons.js

```diff
@@ -1353,11 +1349,24 @@ var createStoreFromObject = function (alt, StoreModel, key, saveStore) {
   return storeInstance;
 };
 
+var checkDispatcherInterface = function () {
+  var dispatcher = arguments[0] === undefined ? {} : arguments[0];
+
+  var requiredMethods = ["waitFor", "dispatchToken", "register", "dispatch"];
```

Comment:
I'm actually not sure if we should even validate the dispatcher interface because there's no way to tell that register and dispatch will work correctly.